hknweb/events/admin.py

```diff
@@ -1,6 +1,43 @@
 from django.contrib import admin
+from django.shortcuts import redirect
 from .models import EventType, Event, Rsvp
 
+class EventAdmin(admin.ModelAdmin):
+
+    fields = ['name', 'slug', 'start_time', 'end_time', 'location', 'event_type', 'description', 'rsvp_limit', 'rsvps', 'created_by', 'created_at']
+    # NOTE: created_by should be read only, but I don't know how to set it to default to current user
```

Comment:
Was this addressed by our earlier messages? created_by should probably be hidden, and set by the POST view function.

Reply:
I think it's useful for VP to be able to see who created it, which is why I wanted it to be read only. However true read only is not possible, so I'll just make it automatically added on the main website, but admins can change it.